src/cmd/add.go

```diff
@@ -26,14 +26,20 @@ var addCmd = &cobra.Command{
 		if err != nil {
 			panic(err)
 		}
-		if !template.HasTemplatesDir(appDir) {
-			fmt.Println("no templates found\n\nPlease add templates to the \".exosphere\" folder of your code base.")
-			os.Exit(1)
+		hasTemplates, err := template.HasTemplatesDir(appDir)
+		if err != nil {
+			panic(err)
```

Comment:
I believe you are misreading this. `hasTemplates` is a boolean for whether or not the user has templates and that is checked against in the next if statement. The error is a possible filesystem error that may be hit when determining this